.github/workflows/open-issue-in-repo.yml

```diff
@@ -78,26 +54,20 @@ jobs:
           echo "inputs.issue_title:           ${{ inputs.issue_title}}"
           echo "inputs.issue_body:            ${{ inputs.issue_body }}"
 
-      # Even thought Github treats issues and PRs as teh same thing behind the scenes, the event context has us access
-      # them in different ways.  On workflow call we want to be able to differentiate between a workflow trigger by
-      # an issue and one trigger by a PR.
+      # Even though GitHub treats issues and PRs as the same thing behind the scenes, the event context accesses
+      # them in different ways.  Choose an issue number of PR number.  Only 1 will be filled.
       - name: "Determine Issue or PR Values"
         id: issue_values
         run: |
-          if [ "${{ github.event_name }}" == "workflow_dispatch" ]; then
-            echo "issue_number=${{ inputs.issue_number }}" >> $GITHUB_OUTPUT
-          else
-            echo "issue_number=${{ github.event.pull_request.number || github.event.issue.number }}" >> $GITHUB_OUTPUT
-          fi
+          echo "issue_number=${{ github.event.pull_request.number || github.event.issue.number }}" >> $GITHUB_OUTPUT
       
-      # this step uses the read permission from the GITHUB_TOKEN it inherits
       - name: "Check for comment"
         uses: peter-evans/find-comment@v3
         id: issue_comment
         with:
           issue-number: ${{ steps.issue_values.outputs.issue_number }}
-          comment-author: "FishtownBuildBot"
           body-includes: "Opened a new issue in ${{ inputs.issue_repository }}:"
+          token: ${{ secrets.FISHTOWN_BOT_PAT }}
```

Comment:
Need to use the PAT so GITHUB_TOKEN doesn't need to be passed in when not using `secrets: inherit`